File: src/main/java/com/gmail/borlandlp/minigamesdtools/conditions/examples/ExampleCondition.java
package com.gmail.borlandlp.minigamesdtools.conditions.examples;

import com.gmail.borlandlp.minigamesdtools.conditions.AbstractCondition;
import org.bukkit.entity.Player;

public class ExampleCondition extends AbstractCondition {
    @Override
    public boolean isValidPlayer(Player p) {
        return true;
    }

    @Override
    public String getErrorId() {
        return "test_error_msg";
    }
}

File: src/main/kotlin/com/gmail/borlandlp/minigamesdtools/conditions/examples/ExampleCondition.kt
package com.gmail.borlandlp.minigamesdtools.conditions.examples

import com.gmail.borlandlp.minigamesdtools.conditions.AbstractCondition
import org.bukkit.entity.Player

class ExampleCondition : AbstractCondition() {
    override fun isValidPlayer(player: Player): Boolean {
        return true
    }

    override val errorId: String
        get() = "test_error_msg"
}